calico_test/Dockerfile

```diff
@@ -11,9 +11,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM jeanblanchard/alpine-glibc
+FROM docker
 MAINTAINER Tom Denham <tom@projectcalico.org>
-
+RUN apk add --update iptables ip6tables iproute2 curl && \
```

Comment:
Stick a comment in as to why we do this, rather than just use the alpine-libc image.